script.js

```diff
@@ -232,13 +232,20 @@ const form = document.querySelector('#form');
 
 function contactValidator(event) {
   const email = document.querySelector('#email').value;
+  const name = document.querySelector('#name').value;
+  const message = document.querySelector('#message').value;
+
   const emailValidation = /[A-Z]/.test(email);
 
   if (emailValidation) {
     emailError.textContent = 'Email Not Sent⚠️, all the letters must be in lower case';
     event.preventDefault();
   } else {
     emailSent.textContent = 'Congratulations, Email sent 👏';
+    localStorage.setItem('name', name);
+    localStorage.setItem('message', message);
+    localStorage.setItem('email', email);
   }
 }
 form.addEventListener('submit', contactValidator);
```

Comment:
- [ ] Kindly make sure form data is updated and saved to the local storage when user changes the content of any input field in the form as requested in the project requirement.

- [ ] Kindly make sure the contact form information is persistent when page is load or reload as requested in the project requirement.

- [ ] Kindly make sure  form information is saved in the local storage, after filling the form and not submitting it.